tests/framework/test_reporting/aggregate_test_results.py

```diff
@@ -136,7 +136,9 @@ def get_best_runs(job_name, build_nums):
     required_tests = set(sys.argv[5].split())
 
     # Fetch the downstream build info from jenkins
-    requests.packages.urllib3.disable_warnings()
+    import urllib3
```

Comment:
Does this need to be imported a second time?

Reply:
shrug... I just changed all the previous usages to this one.